### PR TITLE
Added Coveralls to GitHub actions workflows.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -55,11 +55,13 @@ jobs:
...
           echo '"trac_db_password": "secret", ' >> conf/secrets.json
           echo '"secret_key": "a"}' >> conf/secrets.json
-          pwd
-          cat conf/secrets.json
       - name: Run tox
         run: |
           python -m tox -e py36-tests
+      - name: Coveralls
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (carltongibson) [inline .github/workflows/tests.yml:63]: Did you add this to the repository secrets? Is `GITHUB_TOKEN` a comprehensible name? 
(Maybe `COVERALLS_GITHUB_TOKEN`? 🤔)

**Review by carltongibson — COMMENTED** [review]
Thanks @felixxm

File: .github/workflows/tests.yml (hunk 1 above)
Comment (felixxm) [reply .github/workflows/tests.yml:63]: GitHub automatically creates a `GITHUB_TOKEN` secret and it's sufficient here, see ["About the GITHUB_TOKEN secret"](https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret).

Comment (carltongibson) [reply .github/workflows/tests.yml:63]: Super. 👍

**Review by carltongibson — APPROVED** [review]

**Pull request merged**